sqlitecluster/SQLite.cpp

```diff
@@ -1284,3 +1278,47 @@ map<uint64_t, tuple<string, string, uint64_t>> SQLite::SharedData::popCommittedT
     _committedTransactions.clear();
     return result;
 }
+
+int SQLite::_wrapSQuery(sqlite3* db, const char* e, const string& sql, SQResult& result, int64_t warnThreshold, bool skipInfoWarn) const
+{
+    uint64_t startTime = 0;
+    if (_isFirstQuery && !skipInfoWarn) {
+        startTime = STimeNow();
+    }
+    int squeryResult = SQuery(db, e, sql, result, warnThreshold, _isFirstQuery || skipInfoWarn);
+    if (_isFirstQuery && !skipInfoWarn) {
+        uint64_t elapsed = STimeNow() - startTime;
+        // Avoid logging queries so long that we need dozens of lines to log them.
+        string sqlToLog = sql.substr(0, 20000);
+        SRedactSensitiveValues(sqlToLog);
+        if ((int64_t)elapsed > warnThreshold) {
+            SWARN("Slow first query (" << elapsed / 1000 << "ms): " << sqlToLog);
```

Comment:
Changing from `Slow query` to `Slow first query` is going to break some tooling, off the top of my head:
- https://github.com/Expensify/Web-Expensify/blob/57a50dc4c31d4846a6f2c94206fbc239235ac81e/_utilities/BugBot/Utils.php#L1673-L1683
- https://github.com/Expensify/Salt/blob/e7c215b046cb895c67d90f5c68ac09c2dce11b96/default/files/expensify/bin/logwatcher#L28

Maybe to avoid updating the tooling we can just update this log to say `Slow query (first query)`?